mat-process-api/V1/Validators/PostInitialProcessDocumentRequestValidator.cs

```diff
@@ -0,0 +1,33 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using FluentValidation;
+using mat_process_api.V1.Boundary;
+
+namespace mat_process_api.V1.Validators
+{
+    public class PostInitialProcessDocumentRequestValidator : AbstractValidator<PostInitialProcessDocumentRequest>
+    {
+        public PostInitialProcessDocumentRequestValidator()
+        {
+            ValidatorOptions.CascadeMode = CascadeMode.StopOnFirstFailure;
```

Comment:
Not sure what this is here for. Need to ask and document in the comment. Why can't we return all the errors at once?

Reply:
Added comment clarifying that. It's different from what you'd expect by just looking at it:
It's used to stop the validator from going deeper into any Rule after first failure. 

It doesn't stop checks on each Rule, it's stops continuous checks within the failing Rule. Needed because it stops from checking the properties of the object, if the check for object's existence has failed.